client/src/services/editReview.js

```diff
@@ -0,0 +1,17 @@
+const editReview = async (reviewId, userId, content) => {
+  try {
+    const response = await fetch(`/api/v1/reviews/${reviewId}`, {
+      method: "PATCH",
+      headers: new Headers({
+        "Content-Type": "application/json",
+      }),
+      body: JSON.stringify({ reviewId, userId, content }),
```

Comment:
the userId is accessible in the back end via `req.user` - the front end should not be telling the back end information that the back end already knows from the passport session